Makefile

```diff
@@ -8,13 +8,15 @@ GOTESTSUM_CMD ?= gotestsum --format ${GOTESTSUM_FORMAT} --no-summary=skipped
 
 lint:
 	golangci-lint run --fix
+	golangci-lint fmt
```

Comment:
do we need it here? I get that this restores behaviour we had with v1, but do we need to maintain that?

Reply:
splitting this in the next commit